bundle/1.0.0-1/hive-operator.v1.0.0-1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,620 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    categories: OpenShift Optional
+    certified: 'false'
+    containerImage: quay.io/openshift-hive/hive:1.0.0
```

Comment:
this should probably be v1.0.0 as well, I recall things can get weird out there in stange places and i think this is consistent with most openshift images

Reply:
ok will change

Reply:
done